azure-ipam/ipam.go

```diff
@@ -29,7 +29,8 @@ type IPAMPlugin struct {
 
 type cnsClient interface {
 	RequestIPAddress(context.Context, cns.IPConfigRequest) (*cns.IPConfigResponse, error)
```

Comment:
I think this would make more sense with the current naming conventions. The only reason I didn't was because they would also look very similar and could confuse people. I'm fine with either way though

Are we also planning on removing the `RequestIAddress` function at some point?